### PR TITLE
Fixed for new API changes

### PR DESCRIPTION
**Pull request by 66Bunz**
Fixed outdated "page" parameter in product sales responses with the new "page_key" parameter.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ requests==2.31.0
 
 # Useful for requests debugging.
 requests-toolbelt==0.9.1
+appengine-python-standard==1.1.4
```
Comment (opsdisk) [inline requirements.txt]: This looks like a Google cloud specific library...like AWS' boto?  If so, it should be removed.  Libraries like pygumroad should be vendor agnostic.

File: setup.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
     install_requires=[
         "requests>=2.24.0",
         "requests-toolbelt>=0.9.1",
+        "appengine-python-standard>=1.1.4"
```
Comment (opsdisk) [inline setup.py]: Remove `appengine-python-standard` lib here as well.

**Comment by opsdisk** [comment]
Thanks for the PR @66Bunz - just had a few minor requests.

**Comment by 66Bunz** [comment]
@opsdisk Fixed it

**Comment by opsdisk** [comment]
Thanks for submitting the PR @66Bunz !

**Pull request merged**